backend/fms_core/templates.py

```diff
@@ -168,6 +169,23 @@
     ],
 }
 
+NORMALIZATION_TEMPLATE = {
+  "identity": {"description": "Template to perform normalization", "file": static("submission_templates/Normalization_v3_10_0.xlsx")},
+  "sheets info": [
+      {
+        'name': 'Normalization',
+        'headers': ['Source Container Barcode', 'Source Container Coord', 'Destination Container Barcode', 'Destination Container Coord',
+                    'Destination Container Name', 'Destination Container Kind', 'Destination Parent Container Barcode',
+                    'Destination Parent Container Coord', 'Source Depleted', 'Volume Used (uL)', 'Volume (uL)',
+                    'Conc. (ng/uL)', 'Conc. (nM)', 'Normalization Date (YYYY-MM-DD)', 'Comment'],
+      },
+  ],
+  "prefill info": [
+    ("Normalization", "Source Container Barcode", "container__barcode"),
```

Comment:
Make me think we may be better to add an optional "source sample name" column to the template for prefilling.